stdlib/regex.gr

```diff
@@ -380,19 +400,39 @@ enum ParsedRegularExpression {
   REAlts(ParsedRegularExpression, ParsedRegularExpression),
   RESequence(List<ParsedRegularExpression>, Bool), // seq elts, needs backtrack
   REGroup(ParsedRegularExpression, Number), // regex, group ID
-  RERepeat(ParsedRegularExpression, Number, Option<Number>, Bool), // regex, min, max (None for infinity), true=non-greedy
+  RERepeat(ParsedRegularExpression,
+  Number,
+  Option<Number>,
+  Bool), // regex, min, max (None for infinity), true=non-greedy
```

Comment:
Looks like we're missing some indentation?

Reply:
Good catch, fixed in next push